src/xref2/component.mli

```diff
@@ -432,9 +432,21 @@ and Substitution : sig
 end
 
 and CComment : sig
+  type nestable_block_element =
```

Comment:
Do we need to redefine this here? It looks identical to `nestable_block_element` in `model/comment.ml`.

Reply:
Oh I see, it's because it's referencing `Label.t` here.